src/style.css

```diff
@@ -1,3 +1,60 @@
 body {
   margin: 0;
+  padding: 0;
+  --color-link: #00acdc;
+}
+
+.header {
+  margin: 0;
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+}
+
+.nav {
+  display: inline-block;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__link {
+  margin-left: 20px;
+  height: 60px;
+  line-height: 60px;
+  font-family: "Roboto", sans-serif;
+  color: #000;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)